style.css

```
@@ -2,75 +2,73 @@ html body {
     margin: 0;
     padding: 0;
     font-family: 'Roboto', sans-serif;
+    max-height: 1024px;
+    max-width: 1440px;  
 }
 
-.header {
+.main-header {
     background-image: url("./girl.jpg");
     background-position: center;
     opacity: 0.8;
-    display: grid;
-    grid-template-columns: 1fr 1fr;
-    grid-template-rows: 1fr 6fr;
-    grid-template-areas:
-    "steps" "navigation" 
-    "hero" "hero";
+    height: 1024px;
+}
+
+.content {
+    max-width: 1150px;
     color: white;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    align-items: center;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+
+.navigation {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.navigation li {
+    list-style: none;
+    float: left;
+    padding: 20px;
 }
 
- .main {
-     display: grid;
-     grid-template-columns: 2fr 1fr;
-     grid-template-rows: 2fr;
-     grid-template-areas:
-     "section" "aside"
-     "aside"  "aside";
-     height: 400px;
- }
-
- .footer {
-     display: grid;
-     grid-template-columns: 1fr;
-     grid-template-rows: 1fr;
-     grid-template-areas: 
-     "information";
- }
-
- .navigation {
-     float: left;
-     list-style: none;
-     text-align: center;
-     margin: 30px;
- }
-
- .navigation-list li {
-    display: inline;
-    margin: 30px;
- }
 
 a {
-    padding: 15px;
+    padding: 10px 20px;
     background-color: tomato;
 }
 
 h2 {
-    font-size: 25px;
+    font-size: 30px;
+    float: left;
+    font-weight: bold;
 }
 
 .steps {
      text-align: left;
-     font-weight: bold;
-     margin: 20px;
-     padding-left: 90px;
+     margin: 10px;    
 }
 
 .hero {
     padding: 100px;
+    padding-left: 10px;
```

Tip: 
padding na górze to nic innego niż:
`padding-top: 100px;
padding-right: 100px;
padding-bottom: 100px;
padding-left: 100px;
`
Następnie nadpisujesz
`padding-left: 10px; `

lepszym podejściem byłoby skorzystanie z: 
```suggestion
    padding: 100px 100px 100px 10px;
```
padding: <top><right><bottom><left> (zgodnie ze wskazówkami zegara, od 12:00 począwszy)